php/example_code/apigateway/Get_BasePathmapping.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * ABOUT THIS PHP SAMPLE => This sample is part of the SDK for PHP Developer Guide topic at
+ * https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/ses-template.html
+ *
+ */
+
+require 'vendor/autoload.php';
+
+use Aws\ApiGateway\ApiGatewayClient;
+use Aws\Exception\AwsException;
+
+//Create a ApiGatewayClient
+$APIGatClient = new Aws\ApiGateway\ApiGatewayClient([
+    'profile' => 'default',
+    'version' => '2015-07-09',
+    'region' => 'us-east-2'
+]);
+
+$basePath = '(none)';

```

Comment:
I had to look this up, but it does [look correct](https://docs.aws.amazon.com/apigateway/api-reference/link-relation/basepathmapping-by-base-path/). Slightly unusual API decision...